### DISCUSSION TITLE
Referrers never shows full URL path, only top-level domain

### DISCUSSION THREAD

**Discussion by joshavant** [plausible/analytics#2501] [Self-Hosted Support]
**Version:** 1.5.0-rc2

Hi all,

When I view my dashboard, Referrers data never shows a full URL path (e.g. 'github.com/joshavant'), it only shows the top-level domain (e.g. 'github.com').

Here's an example (this is what I see after I click on 'GitHub' in the 'Sources' module):
<img width="548" alt="Screenshot 2022-12-04 at 9 37 31 PM" src="https://user-images.githubusercontent.com/830519/205544117-eaf984a3-28a1-486e-81c1-9dcba3350a28.png">

This occurs for all sources, including Twitter and any other domains.

The only relevant hosting information that I can think to include is that my server is hosted on AWS Lightsail and I'm using AWS API Gateway as an HTTPS proxy to the static IP of the Lightsail instance. The configuration is entirely default.

**Comment by lightson** [comment]
This could be due to github's ReferrerPolicy.

You could confirm it by clicking on a link to your website from GitHub with dev tools active and check document.referrer in the console.

**Reply by joshavant** [reply]
@lightson If so, why does the public Plausible demo show this information? https://plausible.io/plausible.io?source=GitHub

**Comment by joshavant** [comment]
Kinda wonder if @ruslandoga might know what to do here. :)

**Reply by ruslandoga** [reply]
Sorry, I wouldn't know :(

**Reply by joshavant** [reply]
No worries @ruslandoga. Maybe @metmarkosaric or @ukutaht might have some insight?

**Reply by metmarkosaric** [reply]
this is pretty normal these days. you can read more here: https://plausible.io/blog/referrer-policy

**Reply by joshavant** [reply]
@metmarkosaric Thanks for that info... I guess what's confusing to me is that the official Plausible demo dashboard shows GitHub referral information, and even for traffic using Chrome >85.

Example: https://plausible.io/plausible.io?source=GitHub&browser=Chrome

**Reply by nref** [reply]
Threw me off today too. It must be the case that the demo is displaying cached or simulated requests.
